[2025-01-04T17:22:31.358+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-04T17:22:31.385+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.def0 scheduled__2025-01-02T17:24:00+00:00 [queued]>
[2025-01-04T17:22:31.389+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.def0 scheduled__2025-01-02T17:24:00+00:00 [queued]>
[2025-01-04T17:22:31.389+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-01-04T17:22:31.395+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): def0> on 2025-01-02 17:24:00+00:00
[2025-01-04T17:22:31.401+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'def0', 'scheduled__2025-01-02T17:24:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_1.py', '--cfg-path', '/tmp/tmp_ca70b01']
[2025-01-04T17:22:31.403+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask def0
[2025-01-04T17:22:31.403+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=225) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-04T17:22:31.404+0000] {standard_task_runner.py:63} INFO - Started process 226 to run task
[2025-01-04T17:22:31.435+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag.def0 scheduled__2025-01-02T17:24:00+00:00 [running]> on host de27b012d882
[2025-01-04T17:22:31.478+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nmashtakov' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='def0' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T17:24:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-02T17:24:00+00:00'
[2025-01-04T17:22:31.479+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-04T17:22:31.487+0000] {logging_mixin.py:188} INFO - Hello World
[2025-01-04T17:22:31.487+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-01-04T17:22:31.487+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-04T17:22:31.493+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=def0, run_id=scheduled__2025-01-02T17:24:00+00:00, execution_date=20250102T172400, start_date=20250104T172231, end_date=20250104T172231
[2025-01-04T17:22:31.505+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-04T17:22:31.515+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-04T17:22:31.516+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
